dart/lib/src/api.dart

```diff
@@ -374,12 +374,12 @@ final class Account {
   ///
   /// Uses the recipient's identity key and one-time key to establish a secure channel.
   Session createOutboundSession({
-    required Curve25519PublicKey identityKey,
+    required Curve25519PublicKey recipientIdentityKey,
```

Comment:
Not sure if we really need to rename this. This is an outbound session so it should be clear that the identity key is from the recipient. I think in other libolm/vodozemac bindings this is also just named identityKey